web/olga/analytics/models.py

```diff
@@ -76,6 +76,24 @@ class InstallationStatistics(models.Model):
     )
 
     @classmethod
+    def get_last_for_this_day(cls, edx_installation_object=None):
+        """
+        :param edx_installation_object: specific installation object.
+
+        :return: None or model from db, that was created today.
+        """
+        try:
+            query = cls.objects.filter(edx_installation=edx_installation_object)
```

Comment:
I would use more precise query with filtering objects related to the certain date.